content/blog/announcing-pulumi-preview-improvments/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: "Announcing Pulumi Preview Improvements"
+date: 2020-10-27
+draft: false
+meta_desc: "Announcing a significant improvement to the pulumi preview experience."
+#meta_image: 
+authors:
+    - paul-stack
+tags:
+    - enhancement
+    - pulumi
+---
+
+Today we are announcing a fairly significant improvement to the Pulumi preview experience when working with Pulumi applications.
+Previously, the experience may have been misleading in that some dependent resources would be seen to needing to a "replace" operation as
+reported during the preview, but when apply ran, the "replace" operation didn't actually need to happen.
+
+<!--more-->
+
+## Change in Behaviour
+
+We can look at a sample Pulumi application that creates a Google GKE cluster. Pulumi then deploys an application into the
+kubernetes cluster.
+
+```typescript
```

Comment:
{{< gist pulumipus 0c6e4ede557600e114885869ea0aeb96 >}}

Please delete the rest of the code block because it's just a wall of code that a reader has to scroll through. I created a github account for Pulumipus where we can post gists and then use Hugo shortcodes to embed them. They advantage is that they only display 10 or so lines of code and has a widget that lets you expand the window if you want to see all the code.

Reply:
Amazing!